source/agora/test/Base.d

```diff
@@ -719,6 +722,58 @@ public class TestAPIManager
         }
     }
 
+    // enroll the client
+    private void enroll(size_t client_idx) {
+        writefln!"Enrolling #%s"(client_idx);
```

Comment:
Those will always print, so no. It should be a `log` call.

Reply:
Removed